app/src/pages/inside/stepPage/modals/editDefectModals/makeDecisionModal/optionsStepForm/itemsList/itemsListBody/itemsListBody.jsx

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright 2021 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ItemHeader } from 'pages/inside/logsPage/defectEditor/itemHeader';
+import { StackTraceMessageBlock } from 'pages/inside/common/stackTraceMessageBlock';
+import classNames from 'classnames/bind';
+import { uniqueId } from 'common/utils';
+import styles from './itemsListBody.scss';
+
+const cx = classNames.bind(styles);
+
+const Log = ({ log }) => (
+  <div className={cx('error-log')}>
+    <StackTraceMessageBlock level={log.level} designMode="dark" maxHeight={70}>
+      <div>{log.message}</div>
+    </StackTraceMessageBlock>
+  </div>
+);
+Log.propTypes = {
+  log: PropTypes.object.isRequired,
+};
+const SimilarItemsList = ({ testItems, selectedItems, selectItem, showErrorLogs }) => {
+  return (
+    <>
+      {testItems.length > 0 &&
+        testItems.map((item, i) => {
+          let composeItem = item;
+          if (i !== 0) {
+            const { itemId: id, itemName: name, issue, patternTemplates } = item;
+            composeItem = {
+              ...item,
+              id,
+              name,
+              issue,
+              patternTemplates,
+            };
+          }
+          const selected =
+            !!selectedItems.find((selectedItem) => selectedItem.itemId === item.itemId) || false;
```

Comment:
`|| false` can be ommited:
`const selected = !!selectedItems.find((selectedItem) => selectedItem.itemId === item.itemId)`;